modules/op-deleting-the-pipelines-component-and-custom-resources.adoc

```diff
@@ -28,5 +28,5 @@ Deleting the CRs will delete the {pipelines-title} components, and all the tasks
 
 [IMPORTANT]
 ====
-Repeat the procedure to find and remove CRs of optional components such as {tekton-chains}, before uninstalling the Operator. If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
+Repeat the procedure to find and remove CRs of optional components such as {tekton-hub}, before uninstalling the Operator. If you uninstall the Operator without removing the CRs of optional components, you cannot remove them later.
```

Comment:
@masauso-lungu Is this component optional or not? Please clarify.

Reply:
> the request was to mention `tekton-results` or `tekton-hub` in place of `tekton-chains`

Yes @Srivaralakshmi , that is one of the two options the SME told me to consider in place of `tekton-chains`